### DISCUSSION TITLE
Secondary MX and soft reject

### DISCUSSION THREAD

**Discussion by FredMa01** [rspamd/rspamd#4438] [General]
Hi,
With RSPAMD, when a message is tagged "soft reject", I noticed that it was bounced on my secondary MX. 
How to avoid this?

Thanks
Fred

**Comment by xpufx** [comment]
You have to provide details about reject messages and the way your secondary mx is set up. Ideally you would have rspamd on the backup too and sync settings so they would act the same.

**Comment by FredMa01** [comment]
I have RSPAMD installed on the 2 servers A and B:
Primary MX field points to A (priority 1)
Secondary MX field points to B (priority 10)
Sending an email to server A works well but as soon as a message is tagged "soft reject", it is sent to server B.
For example, here is the rejection message:

```
R_MIXED_CHARSET (1.25) [subject]
ARC_SIGNED (0) [fm2.domain.fr:s=2024:i=1]
DKIM_TRACE (0) [example.org:+]
HAS_REPLYTO (0) [example.org]
CLAM_VIRUS_FAIL (0) [failed to scan and retransmits exceed]
MID_RHS_MATCH_FROM (0)
GREYLIST (0) [greylisted,Mon, 19 Jun 2023 10:06:49 GMT,new record]
MIME_TRACE (0) [0:+]
ARC_NA (0)
RCPT_COUNT_ONE (0) [1]
FROM_HAS_DN (0)
RCVD_COUNT_ZERO (0) [0]
TO_DN_NONE (0)
REPLYTO_ADDR_EQ_FROM (0)
TO_MATCH_ENVRCPT_ALL (0)
FROM_EQ_ENVFROM (0)
ASN (0) [asn:16276, ipnet:myip/16, country:FR]
MIME_GOOD (-0.1) [text/plain]
R_SPF_ALLOW (-0.2) [+a]
R_DKIM_ALLOW (-0.2) [example.org:s=2019]
DMARC_POLICY_ALLOW (-0.5) [example.org,none]
```
